src/app/split-view/multi/terra-multi-split-view.component.scss

```diff
@@ -258,7 +258,7 @@
     height: 525px;
 }
 
-/deep/ .tab-content .fadeIn
+/deep/ .side-scroller .fadeIn
```

Comment:
Werden die Ebenen der Splitview hier nich animiert?

Reply:
Das ist für den Fullscreen. Das div .tab-content wurde verschoben, damit die tabbar vom tab-content getrennt ist und einzeln verborgen werden kann. Deswegen musste die Animation auf ein anderes Element in der Splitview gesetzt werden